core/docker/compose.sh

```diff
@@ -1,3 +1,5 @@
+#!/bin/bash
```

Comment:
Looks like the permissions got changed from 755 to 644 on this file. I think that needs to be reverted. Seems like the execute flag got removed.